server/app/routes/api/orders.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+
+require('../../../db/models/order');
+
+var Order = mongoose.model('Order');
+
+router.get('/', function (req, res, next) {
+	Order.find()
+	.then(function(orders) {
+		res.json(orders);
+	})
+	.then(null, next);
```

Comment:
There is - we haven't defined any custom error handling for these routes, but there's a error handler in top-level index route